packages/button/index.js

```diff
@@ -1,5 +1,7 @@
-const css = require('./css')
-const React = require('./react')
-const vars = require('./vars')
+import Button from './esm/react/index.js'
+import * as vars from './esm/vars/index.js'
+import css from './esm/css/index.js'
 
-module.exports = { css: css, react: React, vars: vars }
+export default Button
+export { vars }
```

Comment:
can the vars and css exports be on the same line?

Reply:
Probably I've never tried that  I'll give it a shot